src/pages/ContactPage/Contact.tsx

```diff
@@ -1,8 +1,299 @@
-import './Contact.scss'
-import { ContactInfos } from './ContactInfos/ContactInfos';
+import './Contact.scss';
+// import './images/e';
+import phone from './images/phone_.svg';
+import email from './images/email.svg';
+import chevron from './images/chevron-right.svg';
+import marker from './images/marker.svg';
+import fb from './images/fb.svg';
+import ins from './images/ins.svg';
+import wp from './images/wp.svg';
+import yt from './images/yt.svg';
+import { useEffect, useState } from 'react';
 
-export const Contact=()=>{
-  return(
-    <ContactInfos/>
+export const Contact = () => {
+  return (
+    <section id="contact">
+      <div className="container">
+        <div className="col-lg-12 col-md-12 col-xs-12">
+          <ul className="first-list">
+            <li>
+              <a href="/">Ana səhifə</a>
+            </li>
+            <li>
+              <img src={chevron} alt="/" />
+            </li>
+            <li className="second-li">
+              <a href="/"> Əlaqə</a>
+            </li>
+          </ul>
+        </div>
+
+        <div className="col-lg-12 col-md-12 col-xs-12">
+          <h1 className="section-title"> ƏLAQƏ</h1>
+        </div>
+        <div className="col-lg-12 col-md-12 col-xs-12">
+          <iframe
+            src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12154.645617216025!2d49.7963393!3d40.3941964!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xb672201e88e0118a!2sGlob.az%20-%20Yeni%20yasamal%20Filial%C4%B1!5e0!3m2!1saz!2s!4v1671106036932!5m2!1saz!2s"
+            loading="lazy"
+            title="map"
+          ></iframe>
+        </div>
+        <div className="col-lg-12 col-md-12 col-xs-12">
+          <div className="contact-info">
+            <h3>Əlaqə məlumatlari</h3>
+            <ul className="contact-list">
+              <li className="mail">
+                <a href="mailto: info@glob.az">
+                  {' '}
+                  <img src={email} alt="/" /> info@glob.az
+                </a>
+              </li>
+              <li className="phone">
+                <a href="/">
+                  {' '}
+                  <img src={phone} alt="/" />
+                  +994 12 525 9001, +994 50 833 9010, +994 99 833 9010
+                </a>
+              </li>
+              <li>
+                <a href="http://127.0.0.1:5500/index.html">
+                  {' '}
+                  <img src={marker} alt="/" />
+                  <b>Azadliq</b> (Ünvan: Binəqədi rayonu, Şövkət Məmmədova küçəsi 2 (Kəmalə Nərmin şadlıq
+                  sarayının arxası))
+                </a>
+              </li>
```

Comment:
you must create a component for such li
this component must take href, imageSrc, boldText and text as props
and you can create an array of objects in your Contact component with all the needed data for you list-items
so you will be able to use array.map() function here